provisioning/roles/shorewall-firewall/files/vpn-shorewall/shorewall.conf

```diff
@@ -227,10 +298,3 @@ PROVIDER_OFFSET=
 MASK_BITS=
 
 ZONE_BITS=0
-
-################################################################################
-#                            L E G A C Y  O P T I O N
-#                      D O  N O T  D E L E T E  O R  A L T E R
-################################################################################
-
-IPSECFILE=zones
```

Comment:
This shows the deleted lines better.

`shorewall update -A` removed this.

Should I put this back in?